src/contact-summary/contact-summary-emitter.js

```diff
@@ -6,21 +6,31 @@ function emitter(contactSummary, contact, reports) {
   var result = {
     cards: [],
     fields: fields.filter(function(f) {
-          if (f.appliesToType === contact.type ||
-              (f.appliesToType.charAt(0) === '!' && f.appliesToType.slice(1) !== contact.type)) {
-            if (!f.appliesIf || f.appliesIf()) {
-              delete f.appliesToType;
-              delete f.appliesIf;
-              return true;
-            }
-          }
-        }),
+      var appliesToType = Array.isArray(f.appliesToType) ? 
+                          f.appliesToType : 
+                          (f.appliesToType ? f.appliesToType.split() : [undefined]);
+      var appliesToNotType = appliesToType.filter(function(type) {
+        return type.charAt(0) === '!';
+      });
+      if (appliesToType.includes(contact.type) ||
+          (appliesToNotType.length > 0 && !appliesToNotType.includes('!' + contact.type))) {
+        if (!f.appliesIf || f.appliesIf()) {
+          delete f.appliesToType;
+          delete f.appliesIf;
+          return true;
+        }
+      }
+    }),
   };
 
   cards.forEach(function(card) {
     var idx1, r, added;
+
+    var appliesToType = Array.isArray(card.appliesToType) ? 
+                        card.appliesToType : 
```

Comment:
is now a good time to make this work with configurable hierarchy?

Reply:
Are you referring to https://github.com/medic/medic-conf/pull/262/files#diff-fb5367cc3c399423b16e5c9057974d8eR6 ?

Reply:
nice! yes